tests/DynamoDbCompositeModelTest.php

```diff
@@ -80,6 +80,79 @@ public function testFindRecord()
         $this->assertEquals($seedName, $item->name);
     }
 
+    public function testFirstOrNew()
+    {
+        DynamoDbModel::unguard();
```

Comment:
Instead of unguarding you could make a base attribute array and extend it per use case?

```
$attributes = ['id' => str_random(), 'name' => 'foo', 'description' => 'bar', 'count' => 0, 'author' => 'foo', 'nested' => [], 'nestedArray' => []];
```